astropy/io/ascii/qdp.py

```
@@ -0,0 +1,559 @@
+            comment_text += line + '\n'
+            continue
+
+        if datatype == "command":
+            # The first time I find commands, I save whatever comments into
+            # The initial comments.
+            if command_lines == "":
+                initial_comments = comment_text
+                comment_text = ""
+
+            if err_specs != {}:
+                warnings.warn(
```

Ruff (0.11.9)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)